### PR TITLE
[Bug] Update main.py to fix toml-lint

### PR DESCRIPTION
**Pull request by bm11100**
## Summary
Toml-lint wasn't working correctly. This small change updates `main.py` and fixes it. 

### Before
```bash
Error loading rule in C
Traceback (most recent call last):
  File "c:\users\bmurphy\appdata\local\programs\python\python39\lib\runpy.py", line 197, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "c:\users\bmurphy\appdata\local\programs\python\python39\lib\runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "C:\dev\Elastic repos\fork-detection-rules\detection-rules\detection_rules\__main__.py", line 34, in <module>
    main()
  File "C:\dev\Elastic repos\fork-detection-rules\detection-rules\detection_rules\__main__.py", line 31, in main
    root(prog_name="detection_rules")
  File "C:\dev\Elastic repos\fork-detection-rules\detection-rules\rulez\lib\site-packages\click\core.py", line 764, in __call__
    return self.main(*args, **kwargs)
  File "C:\dev\Elastic repos\fork-detection-rules\detection-rules\rulez\lib\site-packages\click\core.py", line 717, in main
    rv = self.invoke(ctx)
  File "C:\dev\Elastic repos\fork-detection-rules\detection-rules\rulez\lib\site-packages\click\core.py", line 1137, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "C:\dev\Elastic repos\fork-detection-rules\detection-rules\rulez\lib\site-packages\click\core.py", line 956, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "C:\dev\Elastic repos\fork-detection-rules\detection-rules\rulez\lib\site-packages\click\core.py", line 555, in invoke
    return callback(*args, **kwargs)
  File "C:\dev\Elastic repos\fork-detection-rules\detection-rules\detection_rules\main.py", line 121, in toml_lint
    rules.load_files(Path(p) for p in rule_file)
  File "C:\dev\Elastic repos\fork-detection-rules\detection-rules\detection_rules\rule_loader.py", line 169, in load_files
    self.load_file(path)
  File "C:\dev\Elastic repos\fork-detection-rules\detection-rules\detection_rules\rule_loader.py", line 160, in load_file
    obj = self._deserialize_toml(path)
  File "C:\dev\Elastic repos\fork-detection-rules\detection-rules\detection_rules\rule_loader.py", line 121, in _deserialize_toml
    with io.open(str(path.resolve()), "r", encoding="utf-8") as f:
FileNotFoundError: [Errno 2] No such file or directory: 'C'
```
### After
image

## Contributor checklist

- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)?
- Have you followed the [contributor guidelines](https://github.com/elastic/detection-rules/blob/main/CONTRIBUTING.md)?

### CODE REVIEW

File: detection_rules/main.py [hunk 1]
```diff
@@ -113,8 +113,8 @@ def name_to_filename(name):
 
 
 @root.command('toml-lint')
-@click.option('--rule-file', '-f', type=click.Path(), multiple=True,
-              help='Optionally specify rule files')
+@click.option('--rule-file', '-f', multiple=True, type=click.Path('r'),
```
Comment (rw-access) [inline detection_rules/main.py:116]: @brokensound77 this look right?
```suggestion
@click.option('--rule-file', '-f', multiple=True, type=click.Path(exists=True),
```

File: detection_rules/main.py [hunk 2]
```diff
@@ -113,8 +113,8 @@ def name_to_filename(name):
...
+              help='Optionally specify a specific rule file only')
```
Comment (rw-access) [inline detection_rules/main.py:117]: this doesn't make sense because we have `--multiple=True`

File: detection_rules/main.py (hunk 1 above)
Comment (brokensound77) [reply detection_rules/main.py:116]: 👍

**Review by brokensound77 — APPROVED** [review]

File: detection_rules/main.py (hunk 2 above)
Comment (bm11100) [reply detection_rules/main.py]: ok so this syntax works for multiple, I've updated the help prompt
```
python -m detection_rules toml-lint -f "C:\dev\Elastic repos\fork-detection-rules\detection-rules\rules\aws\defense_evasion_cloudtrail_logging_suspended.toml" -f "C:\dev\Elastic repos\fork-detection-rules\detection-rules\rules\aws\defense_evasion_configuration_recorder_stopped.toml"
```

**Review by rw-access — APPROVED** [review]

**Pull request merged**